### ISSUE TITLE
Cookie management UI

### ISSUE THREAD

**Issue by BenWestcott** [luakit/luakit#967]
**Current Behavior:**
There are built-in UIs for managing settings, ad-blocking lists, bookmarks, and others, but there is no UI for managing cookies. This is more bothersome considering it's stored as a SQL database, which will sometimes require the user to install another program just for this purpose.

**Desired Behavior:**
I can go to luakit://cookies to view and delete cookies individually as well as delete by domain name or clear all cookies.

**Comment by taobert** [comment]
Yes, this would be nice, and it's good to have an open ticket for it, but you shouldn't hold your breath waiting for someone else to implement it. Luakit doesn't do much with cookies itself, it just passes the name of the file to webkit. So, whereas the other pages you mention are built in lua, if a cookies page were possible at all, (i think) it would require c work, and directly interfacing with webkit-gtk.

A pull request would probably be welcomed.

**Comment by MuhammedZakir** [comment]
Until a UI is implemented, you can use Web Inspector. You can only edit cookie (and other storages) for the currently focused domain, but better than nothing I guess.

**Comment by Nomarian** [comment]
If anybody implements this, you might as well implement cookie autodeleting. which would be simpler because all cookies are accepted, but if you don't manually pick a (sub)domain to be kept, it will be erased after.

**Comment by c0dev0id** [comment]
Deleting cookies is possible via https://github.com/luakit/luakit/pull/818

The "UI" is menu driven and could be improved.
